server/build/plugins/json-pages-plugin.js

```diff
@@ -0,0 +1,32 @@
+export default class JsonPagesPlugin {
+  apply (compiler) {
+    compiler.plugin('after-compile', (compilation, callback) => {
+      const pages = Object
+        .keys(compilation.assets)
+        .filter((filename) => /^bundles\/pages.*\.js$/.test(filename))
+
+      const jsonPages = Object
+        .keys(compilation.assets)
+        .filter((filename) => /^bundles\/pages.*\.json$/.test(filename))
+
+      // Delete existing json pages
+      // Otherwise, we might keep JSON pages for deleted pages.
+      jsonPages.forEach((pagePath) => delete compilation.assets[pagePath])
```

Comment:
I think we don't need this since `compilation` instance is recreated on each build. So I think there is no json pages remaining ?